Fix 1.0.0 build

davidhewitt opened the pull request:
Adds another commit upon #15, includes updates to the dependencies in the `meta.yaml`.

regro-cf-autotick-bot committed:
updated v1.0.0

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.14.3, and conda-forge-pinning 2021.11.20.19.20.32

davidhewitt committed:
update recipe requirements for 1.0.0

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

davidhewitt commented:
@nicoddemus I'd be very grateful if you're able to review & merge this today; I'd like to use setuptools-rust 1.0 at work tomorrow via conda.

(Also let me know if you'd like me to join as a maintainer of this feedstock - I think it would make sense given I'm a maintainer of `setuptools-rust` itself!)

nicoddemus committed:
Add davidhewitt as maintainer

As requested in https://github.com/conda-forge/setuptools-rust-feedstock/pull/16#issuecomment-981024129

nicoddemus commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.15.0, and conda-forge-pinning 2021.11.27.20.04.21

nicoddemus commented:
Thanks for fixing it, had not have the time to get to it myself!

> Also let me know if you'd like me to join as a maintainer of this feedstock - I think it would make sense given I'm a maintainer of setuptools-rust itself!

Sure thing, thanks for volunteering. Added you to the maintainer list myself. 👍
